packages/eslint/src/generators/utils/eslint-file.ts

```diff
@@ -64,7 +64,11 @@ export function isEslintConfigSupported(tree: Tree, projectRoot = ''): boolean {
   if (!eslintFile) {
     return false;
   }
-  return eslintFile.endsWith('.json') || eslintFile.endsWith('.config.js');
+  return (
```

Comment:
Should we also include `mjs` here? Use regex instead to match all three.

Reply:
Let's support MJS in a separate PR